### DISCUSSION TITLE
Ubuntu and Npm version change

### DISCUSSION THREAD

**Discussion by BarishSarac** [actions/virtual-environments#4423] [Installed software]
Hello,

Do you guys know when will you update to the new 21.10 Ubuntu version and Npm 7 or 8? This is still using npm 6...

**Comment by miketimofeev** [comment]
Hi!
We don't support non LTS versions of Ubuntu thus the next supported version will be 22.04.
Npm will be updated along with the default nodejs version, probably at the end of November — we will announce it.
